x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -137,6 +142,95 @@ function UserDetailsEditor({ user }: { user: AuthenticatedUser }) {
   );
 }
 
+function UserSettingsEditor({ formik }: { formik: ReturnType<typeof useUserProfileForm> }) {
+  if (!formik.values.data) {
+    return null;
+  }
+
+  return (
+    <EuiDescribedFormGroup
+      fullWidth
+      fieldFlexItemProps={{ style: { alignSelf: 'flex-start' } }}
+      title={
+        <h2>
+          <FormattedMessage
+            id="xpack.security.accountManagement.userProfile.userSettingsTitle"
+            defaultMessage="User Settings"
+          />
+        </h2>
+      }
+      description={
+        <FormattedMessage
+          id="xpack.security.accountManagement.userProfile.themeFormGroupDescription"
+          defaultMessage="Choose your desired theme for Kibana"
+        />
+      }
+    >
+      <EuiFlexGroup responsive={false}>
+        <EuiFlexItem>
+          <FormRow
+            name="data.userSettings.darkMode"
+            label={
+              <FormLabel for="data.userSettings.darkMode">
+                <FormattedMessage
+                  id="xpack.security.accountManagement.userProfile.userSettings.theme"
+                  defaultMessage="Theme"
+                />
+              </FormLabel>
+            }
+            fullWidth
+          >
+            <EuiButtonGroup
```

Comment:
In my older designs, I had suggested the use of a selectable `EuiKeyPadMenu` component, instead of an `EuiButtonGroup`. I liked that direction, as it emphasized the moon/sun icons and made it very easy to identify in a quick scan. Would it be possible to explore that direction here (either now or in the future)?

Reply:
I like the EuiKeyPadMenu too! I had seen the original designs, but wasn't sure what that component was called/how to find it if it existed 😅 

I have a few blockers on this PR, if I can get through them before FF, I will try to change it. If not, definitely in the near future